jscomp/bsb_exe/rescript_main.ml

```diff
@@ -26,7 +26,7 @@ let () = Bsb_log.setup ()
 
 let separator = "--"
 
-let watch_mode = ref false
+let no_deps_mode = ref false
```

Comment:
The first build should be done with dependencies. Builds on change should be without dependencies.